packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -112,6 +131,7 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
         resolvedUrl: IResolvedUrl,
         logger?: ITelemetryBaseLogger,
     ): Promise<IDocumentService> {
+        console.debug(resolvedUrl);
```

Comment:
Please no console logs. If you need to log something, please use telemetry logger.
debug() is Ok solution while we do not have verbose category in telemetry (though I'd add it instead of using debug())

Reply:
Ope, good catch. Leftover from local testing. Thanks!